Spotkanie3_Zadania/CurrencyConversion/CurrencyConversion/Program.cs

```diff
@@ -0,0 +1,172 @@
+                        break;
+                    default:
+                        Console.WriteLine($"No given {foreignCurrency} currency in our exchange office.");
+                        isItGoodCurrency = false;
+                        break;
+                }
+                if (isItGoodCurrency) break;
+                else
+                {
+                    Console.WriteLine("Try again, you can choose from among: \n-Dollar,\n-Euro,\n-Czech crown.");
+                    isItGoodCurrency = true;
+                }
+            }
+            
+            return exchangeRate;
+        }
+
+        static decimal SellCurrency()
+        {
+            bool isItGoodCurrency = true;
+            while (true)
+            {
+                Console.Write("Dear user, tell me what currency you want to sell: ");
+                foreignCurrency = Console.ReadLine();
+                switch (char.ToUpper(foreignCurrency[0]) + foreignCurrency.Substring(1))
+                {
+                    case "Dollar":
+                        exchangeRate = 4.3888m;
+                        break;
+                    case "Euro":
+                        exchangeRate = 4.8349m;
+                        break;
+                    case "Czech crown":
+                        exchangeRate = 0.2470m;
+                        break;
+                    default:
+                        Console.WriteLine($"No given {foreignCurrency} currency in our exchange office.");
+                        isItGoodCurrency = false;  
+                        break;
+                }
+                if (isItGoodCurrency) break;
+                else
+                {
+                    Console.WriteLine("Try again, you can choose from among: \n-Dollar,\n-Euro,\n-Czech crown.");
+                    isItGoodCurrency = true;
+                }
+            }
+            
+            return exchangeRate;
+        }
+
+        static decimal UserCurrency(string foreignCurrency)
+        {
+            Console.Write("Dear user, tell me what currency you want to sell: ");
+            foreignCurrency = Console.ReadLine();
+            Console.Write($"Dear user, give me the exchange rate at which I should count the {foreignCurrency}: ");
+            while (true)
+            {
+                if (decimal.TryParse(Console.ReadLine(), out exchangeRate)) break;
+                else Console.WriteLine($"This is not a number: {exchangeRate.ToString()}. Try again, please.");
+            }            
+            return Math.Round(exchangeRate, 4);
+        }
+
+        static decimal DownloadAmount()
+        {
+           
+            while (true)
+            {
+                if (decimal.TryParse(Console.ReadLine(), out amount)) break;
+                else Console.WriteLine($"This is not a number: {amount.ToString()}. Try again, please.");
+            }
+            return Math.Round(amount, 2);
+        }
+    }
+}
+
+
+
+
```

Comment:
Duzo pustych linii

Reply:
O! widzisz na to nawet nie zwracam uwagę xD